docker/cpp-sysroot.dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM faasm/llvm:13.0.1 as llvm
+# llvm image is not re-built often, so the tag may be behind
+FROM faasm/llvm:0.2.2 as llvm
```

Comment:
This image was tagged with the LLVM image used to build it (i.e. used to be LLVM 13.0.1). It makes it clearer which LLVM version is being used IMO

Reply:
When updating the PR description, I removed the explanation of this change.

Turns out that installing LLVM (inc. libc) populates the sysroot with files (e.g. header files) that are not necessarily removed when re-building libc from scratch. Thus, different `libc` configurations for the same `llvm` version would yield different container images.

Even though adding the `--purge` flag to `llvm.libc` _should_ do the trick, I am not enitrely sure. Thus, rather than having docker image inconsitencies I change the tagging mechanism.